textproc/vectorscan/Portfile

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           boost 1.0
+PortGroup           legacysupport 1.1
+
+# strnlen
+legacysupport.newest_darwin_requires_legacy 10
+
+github.setup        VectorCamp vectorscan 5.4.11 vectorscan/
+revision            0
+categories          textproc
+license             BSD
+maintainers         nomaintainer
+github.tarball_from archive
+supported_archs     arm64 x86_64
```

Comment:
@essandess Do we know why a “portable” fork fails to be portable?

Reply:
It's more portable than hyperscan which only supported x86_64. Per the readme:

> Currently ARM NEON/ASIMD and Power VSX are 100% functional. ARM SVE2 support is in ongoing with access to hardware now. More platforms will follow in the future. Further more, starting 5.4.12 there is now a SIMDe port, which can be either used for platforms without official SIMD support, as SIMDe can emulate SIMD instructions

Reply:
Thank you, interesting. Possibly, next version will work on powerpc.

P. S. Perhaps of little interest for Macports, but “Power VSX are 100% functional” implies that it should work on ppc64, though on cpus with ISA 2.07 (or something around that), so macOS won’t, but BSD and Linux on supported hardware may.